server/hl7/hl7.controller.js

```diff
@@ -104,11 +104,32 @@ function getUserFiles(req, res, next) {
   if (req.user.files.length === 0) {
     err = new APIError('User has no files uploaded', httpStatus.NO_CONTENT);
   } else {
-    err = new APIError('There was an error retrieving uploaded files', httpStatus.BAD_REQUEST);
+    err = new APIError(
+      'There was an error retrieving uploaded files',
+      httpStatus.BAD_REQUEST
+    );
   }
   return next(err);
 }
 
+function getParsedMessages(req, res, next) {
```

Comment:
With no error check here, making a `GET` request to an endpoint like: `http://localhost:4040/api/hl7/files/parsedMessages/5` will cause the server to crash. Maybe we should add some error checking here to ensure that was what passed is actually a mongoose ID. You can do something like this to check:

```
if (mongoose.Types.ObjectId.isValid(ID)) 
```